api/service/staking/service.go

```diff
@@ -30,6 +30,8 @@ const (
 	WaitTime = 5 * time.Second
 	// StakingContractAddress is the staking deployed contract address
 	StakingContractAddress = "TODO(minhdoan): Create a PR to generate staking contract address"
+	// StakingAmount is the amount of stake to put
+	StakingAmount = 10 ^ 18
```

Comment:
this is xor operation. 10^18 .= 24.

if you want 10**18, you should use uint64(10e18)

Reply:
Already changed to simply 10. And the flow is now working.